decidim-core/config/locales/en.yml

```diff
@@ -157,6 +157,7 @@ en:
     filters:
       linked_classes:
         all: All
+        dummyresource: Dummy resources
```

Comment:
Is this key correctly not including a hyphen? Nice decoupling of the `decidim-results` dependency for tests, by the way.

Reply:
Looks like it is!